examples/default/addons.yaml

```diff
@@ -623,7 +623,7 @@ spec:
             # chosen from this range. Changing this value after installation will have
             # no effect. This should fall within `--cluster-cidr`.
             - name: CALICO_IPV4POOL_CIDR
-              value: "${CLUSTER_CIDR}"
+              value: "192.168.0.0/16"
```

Comment:
why is this being hardcoded to `192.168.0.0/16` ?

Reply:
clusterctl does not currently template the addons, so for ease of UX, reset back to the Calico default. Doesn't stop advanced users or downstream consumers from doing what they need to do.